### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.DocSearch-Button-Placeholder)

### ISSUE THREAD

**Issue by A11Ybot-OP** [sass/sass#3228]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Sass: Sass Basics](https://sass-lang.com/guide)
#### Element path
.DocSearch-Button-Placeholder
#### Snippet
`<span class="DocSearch-Button-Placeholder">Search</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.27 (foreground color: #969faf, background color: #ebedf0, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
